spec/helpers/firm_helper_spec.rb

```diff
@@ -54,4 +54,31 @@
       it { is_expected.to eq('2.0 miles away') }
     end
   end
+
+  describe 'minimum_pot_size_text' do
+    let(:available_ordinals) { I18n.t('investment_size.ordinal').keys.map(&:to_s).map(&:to_i) }
```

Comment:
Sure, will do. To note, I could not simply stub `.friendly_name` as the instance available here in the test and the instance in the implementation are different. But the test is shielded from breaking due to copy changes.

It will break only if there are no translations for any InvestmentSize records, or possibly if the default sort order in InvestmentSize stops being `order ascending`. I will document the assumption about sort order.